pandas/core/indexes/base.py

```diff
@@ -6295,6 +6328,22 @@ def map(self, mapper, na_action: Literal["ignore"] | None = None):
             The output of the mapping function applied to the index.
             If the function returns a tuple with more than one element
             a MultiIndex will be returned.
+
+        Examples
+        --------
+        >>> idx = pd.Index([1, 2, 3])
+        >>> idx.map({1: 'a', 2: 'b', 3: 'c'})
+        Index(['a', 'b', 'c'], dtype='object')
+
+        Using `map` with a function:
+
+        >>> idx = pd.Index([1, 2, 3])
+        >>> idx.map('I am a {}'.format)
+        Index(['I am a 1', 'I am a 2', 'I am a 3'], dtype='object')
```

Comment:
😆 nice example

Reply:
There was something similar already in the docs. I stole it.  :-)